Makefile

```
@@ -34,8 +34,25 @@ check-fmt: ## Verify formatting
 markdownlint: ## Lint Markdown files
 	find . -type f -name '*.md' -not -path './target/*' -print0 | xargs -0 $(MDLINT)
 
-nixie: ## Validate Mermaid diagrams
-	find . -type f -name '*.md' -not -path './target/*' -print0 | xargs -0 $(NIXIE)
+nixie: ## Render Mermaid diagrams from .mmd files (writes .svg next to sources)
```

checkmake (0.2.2)

[warning] Target body for "nixie" exceeds allowed length of 5 (18).

(maxbodylength)